APPS_FOR_TESTING.ts

```diff
@@ -0,0 +1,6 @@
+export const APP_NAMES = {
```

Comment:
I think I'd rather have this in the `test.ts` module along with all other test-related stuff.

Reply:
Moved to `test.ts`